.env

```diff
@@ -2,6 +2,6 @@ APP_ID=app.covidshield
 APP_VERSION_CODE=1
 APP_VERSION_NAME=1.0
 TEST_MODE=true
-SUBMIT_URL=https://submit.covidshield.app
-RETRIEVE_URL=https://retrieve.covidshield.app
+SUBMIT_URL=http://localhost:8000
+RETRIEVE_URL=http://localhost:8001
```

Comment:
What I am doing in local development is this:
- Cop `.env` to `.env.local`.
- Change whatever config that I prefer.
- Run native app with:
  + ENVFILE=.env.local yarn run-android.
  + ENVFILE=.env.local yarn run-ios.

So, let's not commit this change.

Reply:
OK. Those URLs aren't going to stay valid forever, but until then, I'll at least update them to their correct values.